README.md

```diff
@@ -1,24 +1,28 @@
 # covid-solver-queue
-## api handlers (old)
-- GET /target
+## API handlers
+**Every POST request checks for API key**
+Pass API key with `curl -X POST -d "apikey=\<apikey>" https://server.domain.com/<handler>`
+
+- POST /target
   Returns target to wich should be all the following requests sent
+  `curl -X POST -d "apikey=\<apikey>" https://server.domain.com/target`
   
-- GET /:target//counter 
+- POST /:target/counter 
   Returns current package number that should be calculated. First we empty the leftover list (filled every two hours with lookup to the ftp server), after all of those are taken, we continue on incrementing active counter.
```

Comment:
We lookup for calculated files directly on server storage without FTP.

Reply:
Fixed